backend/package.json

```diff
@@ -135,7 +135,7 @@
 				},
 				"ApplicationWizard.installationLocation": {
 					"type": "string",
-					"markdownDescription": "Install generators in a specified location. If not set, global location is used."
+					"markdownDescription": "Set the default folder in which generators will be installed. The path to this folder must be valid and absolute. If there is no path defined, the global location will be used."
```

Comment:
With this solution we need to mention that the path must exists

Reply:
approved by Paola.
must be valid --> means it's exist. (valid)